src/status_im/contexts/wallet/common/utils/wallet_connect.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im.contexts.wallet.common.utils.wallet-connect
+  (:require ["@walletconnect/core" :refer [Core]]
+            ["@walletconnect/web3wallet" :refer [Web3Wallet]]
+            [promesa.core :as promesa]
+            [status-im.config :as config]))
+
+(defn- wallet-connect-core
+  []
+  (Core. (clj->js {:project-id config/wallet-connect-project-id})))
```

Comment:
are we going to have a different project-id for testnet mode?

Reply:
Updated